APCAppCore/APCAppCore/Startup/APCAppDelegate.m

```diff
@@ -75,7 +75,7 @@
 static NSString*    const kDemographicDataWasUploadedKey    = @"kDemographicDataWasUploadedKey";
 static NSString*    const kLastUsedTimeKey                  = @"APHLastUsedTime";
 static NSString*    const kAppWillEnterForegroundTimeKey    = @"APCWillEnterForegroundTime";
-static NSUInteger   const kIndexOfProfileTab                = 3;
+static NSUInteger   const __unused kIndexOfProfileTab                = 3;
```

Comment:
I think just remove this, it's incorrect now anyway with the introduction of the news feed tab...